tests/unit/AuditLogApiTest.php

```diff
@@ -126,7 +126,7 @@ public function testCreateAccountLevelLogRecord()
                 ],
                 'exceptions' => false,
                 'json' => [
-                    'actionTime' => '2009-02-13T23:31:30Z',
+                    'actionTime' => '2009-02-14T01:31:30Z',
```

Comment:
Just curious, why?

Reply:
We used `date` and it doesn't convert to UTC, it takes into consideration daylight shifts. Now we use `gmdate` which returns Greenwich time (GMT+0 = UTC)